src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml

```diff
@@ -53,7 +53,7 @@ Actors:
 
 - Name: LoadInitialData
   Type: MonotonicSingleLoader
-  Threads: 50
+  Threads: 1
```

Comment:
I think having just a single thread inserting 100000 documents is probably enough.

Reply:
It looks like resharding workloads use 100 threads to insert 100000 documents. https://github.com/mongodb/genny/blob/bfaff11cabfa9e9f362458f597943259e754f1f1/src/workloads/sharding/ReshardCollection.yml#L87 So I think if we want to go with 100000 documents, we should not use just one thread. Alternatively, we can decrease the number of documents to 10000.

Reply:
From my understanding using 100 threads with a doc count of 100000 inserts 100000 * 100 docs. I reduced the number of documents to 10000 just so the test doesnt take as long to run and should still give meaningful perf data.